assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,11 +56,13 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
         post: storyId,
       });
-      if (elementId) {
-        // TODO Update all elements that use video.
-        const newState = { resource: { ...resource, posterId, poster } };
-        setProperties(elementId, newState);
-      }
+      const newState = {
```

Comment:
Where does resource come from here?

Reply:
Checking, this still clears out the resource. 
![Screenshot from 2020-04-01 12-07-41](https://user-images.githubusercontent.com/237508/78130585-7ab26880-7411-11ea-8485-bb3111a5e77a.png)

Reply:
The updater fixed!